public/app/plugins/datasource/prometheus/add_label_to_query.ts

```diff
@@ -1,10 +1,12 @@
 import _ from 'lodash';
 
-const keywords = 'by|without|on|ignoring|group_left|group_right|bool|or|and|unless';
+const keywords = 'by|without|on|ignoring|group_left|group_right|bool';
+const logicalOperators = 'or|and|unless';
```

Comment:
I have split `logicalOperators` from `keywords` as `logicalOperators` are usually between 2 metrics which is different than other keywords.